resources/authgear/templates/en/translation.json

```diff
@@ -1114,6 +1114,7 @@
   "v2.page.select-account.default.title": "Log in to {AppOrClientName}",
   "v2.page.select-account.default.use-another-account": "Use another account",
   "v2.page.settings-change-password.default.navbar-title": "Change Password",
+  "v2.page.settings-identity-list-email.default.navbar-title": "Email",
```

Comment:
Please also add translation in zh-hk and zh-tw. 

en, zh-hk and zh-tw need us to manually add the translations